packages/create-catalyst/src/utils/check-storefront-limit.ts

```diff
@@ -0,0 +1,33 @@
+import chalk from 'chalk';
+
+import { type BigCommerceChannelsV3Response, type BigCommerceStoreInfo } from './https';
+
+export const checkStorefrontLimit = (
+  channels: BigCommerceChannelsV3Response,
+  storeInfo: BigCommerceStoreInfo,
+) => {
+  const { data: availableChannels } = channels;
+  const { features } = storeInfo;
+
+  if (availableChannels.length >= features.storefront_limits.total_including_inactive) {
+    console.error(
+      chalk.red('\nYou have reached the maximum number of storefronts allowed for your plan.\n'),
+    );
+
+    return false;
+  }
+
+  const activeChannels = availableChannels.filter((channel) => channel.status === 'active');
+
+  if (activeChannels.length >= features.storefront_limits.active) {
+    console.error(
+      chalk.red(
+        '\nYou have reached the maximum number of active storefronts allowed for your plan.\n',
```

Comment:
What's my next step?

(Purchase additional storefront seats in BC control panel or de-activate a channel)

Could even provide a link to channel manager.